### PR TITLE
Shallow Classifiers and tflite models that are ready to run. 

### PR DESCRIPTION
**Pull request by manasmgkar**
This Commit Contains 

1. Ready to use Shallow classifiers trained for major currencies based on the shallow_classifier.h5 from this repo.
2. Ready to Use Tflite shallow currency classifier encoder/decoder fused models that can run on mobile/edge devices using tensorflowlite,
3. Labels for popular currencies so that users can use the labels to create their own encoder-decoder models using the shallow_classifier.h5

I have utilized the shallow classifiers and fused models in a accessibility app i have built for neurodiverse folks since the seeing ai app lacks multiple features useful for neurodiverse folks while interacting with money , the app  -->  https://github.com/manasmgkar/ND-Money , credits to the original repo and researchers here --> https://github.com/manasmgkar/ND-Money#-special-acknowledgements . 

Wanted to contribute them to the main repo so that people can directly use the pretrained shallow_classifiers for most currencies.

### CODE REVIEW

**Comment by manasmgkar** [comment]
@microsoft-github-policy-service agree
